doc/api/child_process.md

```diff
@@ -109,39 +126,36 @@ When running on Windows, `.bat` and `.cmd` files can be invoked using
 spaces it needs to be quoted.
 
 ```js
-// On Windows Only...
 const { spawn } = require('node:child_process');
-const bat = spawn('cmd.exe', ['/c', 'my.bat']);
+const spawned = spawn('my_command', ['arguments']);
 
-bat.stdout.on('data', (data) => {
+spawned.stdout.on('data', (data) => {
   console.log(data.toString());
 });
 
-bat.stderr.on('data', (data) => {
+spawned.stderr.on('data', (data) => {
   console.error(data.toString());
 });
 
-bat.on('exit', (code) => {
+spawned.on('exit', (code) => {
   console.log(`Child exited with code ${code}`);
 });
 ```
 
-```js
-// OR...
-const { exec, spawn } = require('node:child_process');
-exec('my.bat', (err, stdout, stderr) => {
-  if (err) {
-    console.error(err);
-    return;
-  }
-  console.log(stdout);
+```mjs
+import { spawn } from 'node:child_process';
+const spawned = spawn('my_command', ['arguments']);
+
+spawned.stdout.on('data', (data) => {
+  console.log(data.toString());
 });
 
-// Script with spaces in the filename:
-const bat = spawn('"my script.cmd"', ['a', 'b'], { shell: true });
-// or:
-exec('"my script.cmd" a b', (err, stdout, stderr) => {
-  // ...
```

Comment:
Why were these removed?

Reply:
These were modified because in the original, because they are no longer recommended by Node.js. Using `spawn` with a batch file (without `shell: true`) isn't possible, so I figured that I'd remove the example while I was updating the file.

Reply:
Rereading it, these should not have been removed, I've undone that.